docs/tools/cli/service/connection-info.rst

```diff
@@ -0,0 +1,211 @@
+``avn service connection-info``
+==================================================
+
+Here you’ll find the full list of commands for ``avn service connection-info``.
+
+
+Retrieve connection information
+--------------------------------------------------------
+
+``avn service connection-info kafkacat``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the ``kcat`` :doc:`command </docs/products/kafka/howto/kcat>` necessary to connect to a certain Aiven for Apache Kafka® service.
```

Comment:
I think "connect to a certain" should be "connect to an" or "connect to a particular"

Reply:
Not sure this belongs here but: why is it `kafkacat` if the tool is called `kcat` ....?

Reply:
I open an issue on Jira regarding that